Doc/c-api/init.rst

```diff
@@ -1091,6 +1091,10 @@ Python-level trace functions in previous versions.
    | :const:`PyTrace_C_RETURN`    | Function object being called.        |
    +------------------------------+--------------------------------------+
 
+   Any trace function registered using :c:func:`PyEval_SetTrace` will not receive
+   :const:`PyTrace_C_CALL`, :const:`PyTrace_C_EXCEPTION`, :const:`PyTrace_C_RETURN`
+   or :const:`PyTrace_LINE` as a value for the *what* parameter.
+
```

Comment:
Why line event is also here?

Reply:
In this same page in the documentation, prior to any change it says that:
>void PyEval_SetTrace( | Py_tracefunc func, PyObject *obj)
>Set the tracing function to func. This is similar to PyEval_SetProfile(), except the tracing function does receive line-number events.

So I included `PyTrace_LINE`. Did I misinterpret this?

Reply:
Sorry, I misread that part.